test/conformance/chainsaw/validate/clusterpolicy/standard/apicalls/default/policy.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: default
+spec:
+  rules:
+  - name: default-apicall
+    match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+          operations:
+          - CREATE
+    context:
+    - name: testString
+      apiCall:
+        urlPath: "/api/v1/namespaces/{{ request.namespace }}/invalid"
+        jmesPath: metadata.name
+        default: default
+    - name: testJSON
+      apiCall:
+        urlPath: "/api/v1/namespaces/{{ request.namespace }}/invalid"
+        default: '{"metadata": {"name": "default"}}'
+    - name: testInteger
+      apiCall:
+        urlPath: "/api/v1/namespaces/{{ request.namespace }}/invalid"
+        jmesPath: metadata.resourceVersion
+        default: 1     
+    validate:
+      deny:
+        conditions:
+          all:
+          - key: "{{ testString }}"
+            operator: Equals
+            value: "{{ request.namespace }}"
+          - key: "{{ testJSON.metadata.name }}"
+            operator: Equals
+            value: "{{ request.namespace }}"
+          - key: "{{ testInteger }}"
+            operator: GreaterThan
+            value: 2
```

Comment:
I think this policy will never deny any request. In both scenarios (pod and bad-pod) the testInteger will always default to 1 and therefore the last conditions will never be true, thus no request will ever get denied by this policy.
or do I understand something wrong?